climate_scanner/trends_innovation_classifier/data_utils.py

```diff
@@ -218,11 +266,18 @@ def data_processing(text:str) -> str:
     Arguments: Str - Free Text Ex: Title / Document / Article etc.
     :return: Str - Cleaned text
     """
-    text = text.lower()
-    text = text.replace(r'&lt;p&gt;', '')
-    text = text.replace(r'&amp;apos;','')
-    text = text.replace(r'<.*?>', '')
-    text = text.replace(r'http\S+', '')
+#     text = text.lower()
+#     text = text.replace(r'&lt;p&gt;', '')
+#     text = text.replace(r'&amp;apos;','')
+#     text = text.replace(r'<.*?>', '')
+#     text = text.replace(r'http\S+', '')
+    text = text.replace(r'\.+', ".")
+    
+    replacement_dict={"e.t.c.":"etc","e.g.":"eg","i.e.":"ie"}
```

Comment:
@Christine - instead of doing a cleanup of abbreviations this way, can we use a regex to clean-up many such occurences. 
regex = r"((?:\w\.){2,})"

If you check this regex using https://regexr.com/ or you can do a re.findall on all our data as well, but this help us many variant of abbreviations with full-stops Eg: B.B.C , Mr., Ms. , e.t.c (without ending full-stop) or any other such occurrences.

Let me know if this makes sense.